### PR TITLE
Add an update_config script to update Prow job config

### LINKED ISSUE #2411: We don't have a way to manually update the Prow job config for Knatve
Currently the Prow job config is updated with a plugin and it's defined in https://github.com/GoogleCloudPlatform/oss-test-infra/blob/master/prow/knative/plugins.yaml#L75-L80. The plugin will update the Prow job config after a new change is submitted and merged in the knative/test-infra repo.

This process normally works pretty well, but sometimes e.g. when Prow has an outage due to misconfigured Prow jobs, we won't be able to merge a PR thus won't be able to trigger the plugin, so we will need a way to manually update the Prow job config. Currently there is not a way to do this in knative/test-infra repo.

cc @chaodaiG @peterfeifanchen

### PR DESCRIPTION
**Pull request by chizhg**
**What this PR does, why we need it**:
Add an update_config script to update Prow job config

**Which issue(s) this PR fixes**:
Fixes #2411

### CODE REVIEW

File: config/prod/prow/update_config.sh [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+source $(dirname $0)/../../../scripts/library.sh
+
+set -e
+
+orig_kubeconfig=${KUBECONFIG}
+KUBECONFIG="/tmp/kubeconfig.yaml"
+make -C "${REPO_ROOT_DIR}/config/prod" get-cluster-credentials
+add_trap "make -C ${REPO_ROOT_DIR}/config/prod unset-cluster-credentials" EXIT
+add_trap "rm ${KUBECONFIG}" EXIT
+add_trap "KUBECONFIG=${orig_kubeconfig}"
```
Comment (chaodaiG) [inline config/prod/prow/update_config.sh:29]: no trap condition?

Comment (chizhg) [reply config/prod/prow/update_config.sh]: Oops missed it. Fixed.

File: config/prod/prow/jobs/config.yaml [hunk 2]
```diff
@@ -17642,10 +17642,10 @@ periodics:
   decorate: true
   cluster: "build-knative"
   extra_refs:
-  - org: knative
+  - org: chizhg
```
Comment (chaodaiG) [inline config/prod/prow/jobs/config.yaml:17645]: sure?

Comment (chizhg) [reply config/prod/prow/jobs/config.yaml]: Removed.

**Pull request merged**